templates/admin/release_report_detail.html

```diff
@@ -349,9 +349,16 @@ <h2 class="text-orange mb-1 mt-0">{{ item.library.name }}</h2>
                   {% endif %}
                 {% endwith %}
                 <div>
-                  There {{ item.issues.opened|pluralize:"was,were" }} {{ item.issues.opened }} issue{{ item.issues.opened|pluralize }} opened.
-                  There {{ item.issues.closed|pluralize:"was,were" }} {{ item.issues.closed }} issue{{ item.issues.closed|pluralize }} closed.
+                  There {{ item.issues.opened|pluralize:"was,were" }} {{ item.issues.opened }} issue{{ item.issues.opened|pluralize }} opened
+                  There {{ item.issues.closed|pluralize:"was,were" }} {{ item.issues.closed }} issue{{ item.issues.closed|pluralize }} closed
```

Comment:
IMO the arrangement you used for dependencies is preferred.

```suggestion
                  There {{ item.issues.opened|pluralize:"was,were" }} {{ item.issues.opened }} issue{{ item.issues.opened|pluralize }} opened
                  and {{ item.issues.closed }} issue{{ item.issues.closed|pluralize }} closed
```